boards/arm/qemu_cortex_a53/qemu_cortex_a53_defconfig

```diff
@@ -18,3 +18,9 @@ CONFIG_ARM_PSCI=y
 CONFIG_UART_PL011=y
 CONFIG_UART_PL011_PORT0=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
+
+CONFIG_SMP=y
```

Comment:
I prefer having a different defconfig for SMP and leaving the `qemu_cortex_a53_defconfig` only for !SMP. This is also considering that the userspace work at beginning will not have support for SMP. Please add a new `qemu_cortex_a53_smp_defconfig`.

Reply:
ok